### PR TITLE
Feat: cee identity_key on simulate

### PR DESCRIPTION
**Pull request by nmrgt**
- add identity_key on simulate endpoint #2171 
- add journey_id and status in case of 409 #2164

### CODE REVIEW

**Comment by jonathanfallon** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit 8bc4939302bfcd35f68fad61ea0fa14b41c48184. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=63ebd539a53ecd331737f524&branchName=2189,feat/cee_id_key_expl)

---


<details>
<summary>✅ Successfully ran 2 targets </summary>

- [`lerna run test:unit`](https://cloud.nx.app/runs/fbBCjkkh7i)
- [`lerna run build`](https://cloud.nx.app/runs/qqY3iCXGaN)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: api/services/cee/src/actions/RegisterCeeAction.ts [hunk 1]
```diff
@@ -128,8 +128,15 @@ export class RegisterCeeAction extends AbstractAction {
           // Server error
           throw new Error('Boum');
         } else {
+          if (operator_id === old.operator_id) {
+            throw new ConflictException({
+              uuid: old.uuid,
+              datetime: old.datetime.toISOString(),
+              journey_id: old.acquisition_id,
```
Comment (P3rceval) [inline api/services/cee/src/actions/RegisterCeeAction.ts:135]: l'`acquisition_id` cela ne correspond ni à notre `journey_id` ni à celui des opérateurs. Est-ce que cela ne pas pas porter à confusion sur le nom du champ pour les opérateurs ?

Comment (nmrgt) [reply api/services/cee/src/actions/RegisterCeeAction.ts:135]: `acquisition_id` = `journey_id` chez nous

**Review by P3rceval — APPROVED** [review]

**Pull request merged**